src/config/dynamic-theme-fixes.config

```diff
@@ -6940,13 +6940,13 @@ img[src*="empty-cart.png"]
 .-gy5 > .-fs0 > .vent-link[title*="Food"]
 .-gy5 > .-fs0 > .vent-link[title*="Party"]
 .-gy5 > .-fs0 > .vent-link[title*="Now"]
-.col4 > nav.s-menu > a.m-itm:not(:nth-child(1)):not(:nth-child(6)) > svg
+.col4 > .s-menu > a:not(:nth-child(1)):not(:nth-child(3)):not(:nth-child(7)) > svg
 div.-fw-w:nth-child(1) > a.-fs0 > img
 article.-df > svg > use
 svg.ic.xprss
-svg.ic[alt*="Express"]
-svg.ic[aria-label*="Express"]
-svg.ic[aria-label*="Global"]
+svg.ic[alt*="جوميا"]
+svg.ic[aria-label*="اكسبريس"]
+svg.ic[aria-label*="جلوبال"]
```

Comment:
This seems like language-independent? Could these be changed to something that doesn't rely on the aria-label?

Reply:
Hmm, indeed it's language-dependent.
The css code is obfuscated, so it was a little annoying trying to get a nice and short selector.